indra/newview/scripts/lua/test_luafloater_demo.lua

```diff
@@ -74,5 +64,3 @@ function process_events(waitfor)
 end
 
 coro.launch(process_events, catch_events)
```

Comment:
Use `fiber.launch()` now, and pass a name. `LuaState::expr()` now detects if a script has imported `fiber.lua`, directly or indirectly, and if so it follows up with an implicit `fiber.run()` call that runs launched coroutines to completion. But `fiber.run()` only knows about coroutines launched with `fiber.launch()`.